addons/mail/views/mail_activity_views.xml

```diff
@@ -332,4 +344,85 @@
         </field>
     </record>
 
+    <record id="mail_activity_open_target_view_tree" model="ir.ui.view">
+        <field name="name">mail.activity.view.tree</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail_activity_view_tree"/>
+        <field name="mode">primary</field>
+        <field name="priority">32</field>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="action">action_open_document</attribute>
+                <attribute name="type">object</attribute>
+                <attribute name="js_class">archive_disabled_activity_list</attribute>
+                <attribute name="multi_edit">1</attribute>
+            </xpath>
+            <xpath expr="//tree//field[@name='activity_type_id']" position="attributes">
+                <attribute name="domain">[('res_model', '=', False)]</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="mail_activity_open_target_view_kanban" model="ir.ui.view">
+        <field name="name">mail.activity.view.kanban</field>
+        <field name="model">mail.activity</field>
+        <field name="arch" type="xml">
+            <kanban string="Activity" action="action_open_document" type="object">
+                <templates>
+                    <t t-name="kanban-box">
+                        <div t-attf-class="oe_kanban_card oe_kanban_global_click d-flex flex-column justify-content-between">
+                            <div class="d-flex justify-content-between">
+                                <span>
+                                    <field name="res_name"
+                                        class="fw-bold" /> (<field class="text-muted" name="res_model_id" />)</span>
+                                <field name="activity_type_id" widget="badge" />
+                            </div>
+                            <div class="d-flex flex-row gap-2">
+                                <div class="text-truncate">
+                                    <field name="summary" />
+                                </div>
+                            </div>
+                            <div class="d-flex justify-content-between align-items-center">
+                                <div class="d-flex gap-2 align-items-center">
+                                    <field name="user_id" widget="many2one_avatar_user" />
+                                    <field name="date_deadline" widget="remaining_days" />
+                                </div>
+                                <div class="d-flex flex-row gap-2">
+                                    <button type="object" name="unlink" class="btn btn-link btn-sm">
```

Comment:
should be name="action_done"?
BTW: shall we add invisible modifier based on 'state' value?